File: src/main/kotlin/com/techeasy/contas/receber/application/HomeController.java
package com.techeasy.contas.receber.application;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;

@Controller
@RequestMapping("/home")
public class HomeController {
    @GetMapping
    String index(){
        return "home/index";
    }
}

File: src/main/kotlin/com/techeasy/contas/receber/application/HomeController.kt
package com.techeasy.contas.receber.application

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping

@Controller
@RequestMapping("/")
class HomeController {

    @GetMapping
    fun index(): String {
        return "index";
    }
}